server/backend/testspg/router/test_router_application.py

```diff
@@ -153,15 +171,15 @@ def test_get_fam_application_user_role_assignment_concrete_role(
 ):
     access_roles_fom_dev_only = ["FOM_DEV_ADMIN"]
 
-    role_assignment_end_point = endPoint + \
-        f"/{TEST_FOM_DEV_APPLICATION_ID}/user_role_assignment"
+    role_assignment_end_point = end_point + \
+        f"/{FOM_DEV_APPLICATION_ID}/user_role_assignment"
```

Comment:
this one as well maybe? same as the `get_application_role_assignment_end_point`?